### DISCUSSION TITLE
Prisma Data Browser 504 Error

### DISCUSSION THREAD

**Discussion by masnwilliams** [prisma/prisma#11850] [Q&A] [answered]
Hello! I am trying to view my data from the prisma data viewer online but whenever I go to load a table, it tries to run for about 20-30 seconds and then fails with the following error:
![image](https://user-images.githubusercontent.com/43387599/154309312-5aa617e4-ab8c-42a5-bd4e-af8c7ada4d16.png)

This just happened the other day and I cannot find any fix for it anywhere. It happens on every table no matter if there is data in  the  table or not.

Any help and suggestions would be greatly appreciated.

**Comment by masnwilliams** [comment]
Adding to this, the local data browser works perfectly fine. It is just the online data browser in the cloud that is the issue. I have ensured that my schema is up-to-date on my github repo which prisma has read and updated the schema to reflect those changes.

**Comment by nurul3101** [comment, marked as answer]
Hey @johnmasonwilliams 👋🏻 ,
Can you please provide your project handle in which you are facing this issue?
You can find the project handle in the URL, please refer the image below:

<img width="506" alt="Screenshot 2022-02-23 at 12 09 36 PM" src="https://user-images.githubusercontent.com/13049676/155272051-d9ede3d5-7ad4-4d1e-a077-2aefd322cab9.png">

In this case, handle is `prisma-data-platform-starter`

**Reply by masnwilliams** [reply]
Hey @nurul3101,

My project handle is `bloxsmith`.

I talked to someone through the website chat box but haven't heard back for 10 days now. This still remains an issue.

I would just like to be kept in the loop if this is an issue that is going to take a while to fix.

**Reply by nurul3101** [reply]
Our team were able to reproduce this issue and are working to fix it. 

It's not been fixed yet, I'll make sure to keep you updated and let you know once this is fixed so you could test it.

We appreciate all your help in making prisma data platform ready for General Availability. 🙌

**Reply by masnwilliams** [reply]
Ok, thank you for keeping me updated.

Always willing to help!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
